.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,59 @@
+name: Feature request
+description: Request a feature or improvement
+labels: ['enhancement']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This form is for requesting features or improvements in MicroPython.
+
+        #### Not a MicroPython core feature?
+
+        * If you have a question \"How Do I ...?\", please post it on [GitHub Discussions](https://github.com/orgs/micropython/discussions/) or [Discord](https://discord.gg/RB8HZSAExQ) instead of here.
```

Comment:
Many feature requests would be better starting off life as a discussion. So maybe an extra line below this one also linking to discussions/discord to suggest "get feedback on your idea first" etc.

Reply:
@jimmo Done, PTAL!